docs/internals/howto-release-django.txt

```diff
@@ -15,7 +15,7 @@ There are three types of releases that you might need to make:
 
 * Security releases: disclosing and fixing a vulnerability. This'll
   generally involve two or three simultaneous releases -- e.g.
-  3.2.x, 4.0.x, and, depending on timing, perhaps a 4.1.x.
+  3.2.x, 4.2.x, and, depending on timing, perhaps a 4.1.x.
```

Comment:
I don't see much value in changing it. Also, updating to `4.2.x` makes the rest of the sentence _"perhaps a 4.1.x"_ incorrect.